pages.pt_BR/osx/icalbuddy.md

```diff
@@ -0,0 +1,24 @@
+# icalBuddy
+
+> Utilitário de linha de comando para exibir eventos e tarefas do banco de dados do calendário do macOS.
+> Mais informações: <https://hasseg.org/icalBuddy/>.
+
+- Exibe eventos que acontecerão hoje:
+
+`icalBuddy -n eventsToday`
```

Comment:
```suggestion
`icalBuddy --includeOnlyEventsFromNowOn eventsToday`
```

Optional change (requires the [original](https://github.com/tldr-pages/tldr/blob/main/pages/osx/icalbuddy.md) page update): long option can be used.

Reply:
Resolved in #9534.